Source/UI/NewUI/CommonUI/View/Core/ButtonView.swift

```diff
@@ -50,19 +55,16 @@ class ButtonView: UIView {
     }
 
     private func updateButtonStyle(with style: ElementButtonStyle) {
+        backgroundColor = isEnabled ? style.backgroundColor : style.disabledTintColor
         button.tintColor = .clear
-        button.clipsToBounds = true
-        button.layer.borderColor = style.normalBorderColor.cgColor
-        button.layer.borderWidth = style.borderWidth
-        button.setTitleColor(style.disabledTextColor, for: .disabled)
-        button.setTitleColor(style.textColor, for: .normal)
+        button.heightAnchor.constraint(equalToConstant: style.height).isActive = true
```

Comment:
Can we do without forcing the height of the button?

Alternatively, we could create a constraint for the height to be "At least 44 pixels height" (in line with HIG) but not actually limit it from growing taller if it so needs to

Reply:
button height  is controlled by the merchant `style.height`. after current release, merchant will be able to change any sizes